docs/azure/design-build-deploy/user-management.md

```diff
@@ -2,76 +2,114 @@
+```
+Project Set Management Group (abc123)
+├── Development Subscription (abc123-dev)
+├── Test Subscription (abc123-test)
+├── Production Subscription (abc123-prod)
+└── Tools Subscription (abc123-tools)
+```
 
-## What you can't do
+All your resources are organized within these subscriptions, and your unique Project Set license plate (e.g., "abc123") prefixes all your management groups and subscriptions.
 
-As a Product Owner or Technical Lead with restricted Owner permissions, you don't have the ability to:
+## Access Management Overview
 
-- Assign users to an Owner role
-- Assign users to roles at various levels:
+### Permission Tiers
 
-   - Project Set Management Group (prefixed with your Project Set license plate, ie. "abc123")
-   - Subscription (prefixed with your Project Set license plate, ie. "abc123")
-   - Resource groups
-   - Individual resources
+Your Project Set uses three standard permission levels:
 
-!!! info "Assigning Users to Roles"
-    We are transitioning from direct user role assignments to Entra ID Security Groups. This simplifies access auditing for regulatory compliance.
+- **Reader**: View-only access to resources
+- **Contributor**: Can create and manage resources but cannot modify access permissions
+- **Owner**: Administrative access with full control (with certain security restrictions)
 
-    You'll still be able to directly assign Service Principals to Roles.
+!!! note "Policy Restrictions"
+    While these roles provide different levels of access, all actions are still subject to Azure Policy restrictions. Even with Owner or Contributor roles, certain operations may be restricted by policies applied to your Project Set. These policies help ensure compliance with security standards and organizational requirements.
 
-## Best practices for user management
+### Security Group Structure
 
-To ensure secure and efficient user management, we recommend the following best practices:
+Access is managed through EntraID security groups that follow this naming pattern:
 
-1. **Assign roles at higher levels**: Whenever possible, assign users to roles at the Management Group or Subscription levels. This approach simplifies management and provides consistent access across resources.
+`DO_PuC_Azure_Live_{LicensePlate}_{Role}`
 
-2. **Follow the principle of least privilege**: Only give users the roles they need to perform their specific job functions. This minimizes potential security risks.
+For example:
 
-3. **Regularly review access**: Periodically review user access and remove unnecessary permissions to maintain a secure environment.
+- `DO_PuC_Azure_Live_abc123_Owners`
+- `DO_PuC_Azure_Live_abc123_Contributors`
+- `DO_PuC_Azure_Live_abc123_Readers`
 
-## How to manage users in the Azure portal
+These groups are assigned roles at the management group level, which automatically propagates permissions down to all four subscriptions. This approach ensures consistent access control across your entire Project Set.
```

Comment:
Should "management group" be capitalized (ie. Management Group)?

Also, you mention 4 subscriptions, but they may not have all 4, so should that line be more generic?